packages/devtools_app/lib/src/extension_points/extensions_external.dart

```diff
@@ -49,44 +49,43 @@ class ExternalDevToolsExtensionPoints implements DevToolsExtensionPoints {
 
 const _newDevToolsIssueUriDisplay = 'github.com/flutter/devtools/issues/new';
 
-Uri _newDevToolsGitHubIssueUri({String? issueDetails}) {
+Uri _newDevToolsGitHubIssueUriLengthSafe({String? issueDetails}) {
```

Comment:
nit: maybe rename issueDetails parameter to `additionalInfo` or similar. The names `issueDetails` and `issueLinkDetails` are similar enough that it makes this tough to follow.